Firestore/core/src/core/filter.h

```diff
@@ -88,8 +92,24 @@ class Filter {
     return rep_->ToString();
   }
 
-  size_t Hash() const {
-    return rep_->Hash();
+  /** Check whether the filter is meaningful */
```

Comment:
`Returns true if and only if the filter is a composite filter that doesn't contain any field filters.`